File: INCERCION EN ARREGLOS/INCERCION.cpp
#include <iostream>
#include <vector>
using namespace std;

int main() {
    vector<int> arreglo = {5, 10, 15, 20, 25, 30};

    cout << "Arreglo original: ";
    for (int num : arreglo) {
        cout << num << " ";
    }
    cout << endl;

    arreglo.push_back(35); 

    cout << "Arreglo después de insertar: ";
    for (int num : arreglo) {
        cout << num << " ";
    }
    cout << endl;

    return 0;
}

File: INCERCION EN ARREGLOS/INCERCION.java
import java.util.ArrayList;

public class INCERCION {
    public static void main(String[] args) {
        ArrayList<Integer> arreglo = new ArrayList<>();
        
       
        arreglo.add(10);
        arreglo.add(20);
        arreglo.add(30);
        arreglo.add(40);
        arreglo.add(50);

        System.out.println("Arreglo original: " + arreglo);

       
        arreglo.add(60);

       
        arreglo.add(2, 25);

        System.out.println("Arreglo después de insertar: " + arreglo);
    }
}
